tests/fixture/slp.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+
+@pytest.fixture
+def slp_file1():
+    # Typical SLP file including Predicted and normal `Instance` objects, tracks, and
```

Comment:
Predicted and normal `Instance` objects, tracks, and skeletons -> `PredictedInstance`, `Instance`, `Track`, and `Skeleton` objects (with backticks)